turbopack/crates/turbopack-ecmascript/src/tree_shake/graph.rs

```diff
@@ -34,7 +34,6 @@ use super::{
 };
 use crate::{magic_identifier, tree_shake::optimizations::GraphOptimizer};
 
-const FLAG_DISABLE_EXPORT_MERGING: &str = "TURBOPACK_DISABLE_EXPORT_MERGING";
```

Comment:
We don't need this flag anymore. I removed the flag with https://github.com/vercel/next.js/pull/77063